### ISSUE TITLE
The 2.0.0 version jar looks empty (no clojure source)

### ISSUE THREAD

**Issue by niwinz** [Olical/depot#44]
Hello!

I have noticed after update to 2.0.0 that the jar does not contaiins clojure/java code.

I first found the following error:

```text
user@main:~/uxbox/frontend/ > clojure -A:dev:outdated
Downloading: olical/depot/2.0.0/depot-2.0.0.pom from clojars
Downloading: olical/depot/2.0.0/depot-2.0.0.jar from clojars
Execution error (FileNotFoundException) at clojure.main/main (main.java:40).
Could not locate depot/outdated/main__init.class, depot/outdated/main.clj or depot/outdated/main.cljc on classpath.

Full report at:
/tmp/clojure-8857071319473603149.edn
```

After looking at the downloaded jar, i found the following:
![depot](https://user-images.githubusercontent.com/843689/86214870-13c15e80-bb7c-11ea-984d-cc5381e3c02f.png)

**Comment by Olical** [comment]
Oh no! I did this from a machine that I haven't deployed from before and it
did have some weird complaints. Thanks for letting me know, I'll fix it.

**Comment by Olical** [comment]
Fixed in `v2.0.1`, my `pom.xml` was slightly wonky.
